examples/kubernetes/konnectivity-server.yaml

```diff
@@ -35,7 +36,7 @@ spec:
       httpGet:
         scheme: HTTP
         host: 127.0.0.1
-        port: 8092
+        port: 8093
```

Comment:
We should probably add health in the ports section (below) as well.

Reply:
done